app/src/main/java/studyBuddy/Session.java

```diff
@@ -27,14 +39,40 @@ public Session() {
         startTime = null;
         endTime = null;
         name = null;
-        expectedTime = 0.0;
+        expectedTime = 0;
         productiveTime = 0.0;
         totalTime = 0.0;
-        // assume they were productive the whole time
         percentProductive = 1.0;
-        expectedTime = 0.0;
-        // timeManager = null;
         sessionOngoing = false;
+        sessionPaused = false;
+
+        // manage session events
```

Comment:
can we make the comments on these a bit longer - it took me a long time to trace them around

Reply:
for sure -- i'll take a pass thru all of this and add some more detailed comments to everything